distrobuilder/main_repack-windows.go

```diff
@@ -230,15 +203,68 @@ func (c *cmdRepackWindows) run(cmd *cobra.Command, args []string, overlayDir str
 	logger.Info("Mounting driver ISO")
 
 	// Mount driver ISO
-	err := shared.RunCommand(c.global.ctx, nil, nil, "mount", "-t", "iso9660", "-o", "loop", virtioISOPath, driverPath)
+	err = shared.RunCommand(c.global.ctx, nil, nil, "mount", "-t", "iso9660", "-o", "loop", c.flagDrivers, driverPath)
 	if err != nil {
-		return fmt.Errorf("Failed to mount %q at %q: %w", virtioISOPath, driverPath, err)
+		return fmt.Errorf("Failed to mount %q at %q: %w", c.flagDrivers, driverPath, err)
 	}
 
+	c.umounts = append([]string{driverPath}, c.umounts...)
```

Comment:
c.umounts = append(c.umounts, driverPath)

Reply:
It's following the rule that first mounted should be last umounted

Reply:
Ah, I see. Your updated way of handling that seems better (process the list in reverse order during unmount).